.ci/azure-pipelines-main.yml

```diff
@@ -43,7 +43,7 @@ jobs:
         displayName: "Build Web Client"
         condition: and(succeeded(), or(contains(variables['System.PullRequest.TargetBranch'], 'release'), contains(variables['System.PullRequest.TargetBranch'], 'master'), contains(variables['Build.SourceBranch'], 'release'), contains(variables['Build.SourceBranch'], 'master')), eq(variables['BuildConfiguration'], 'Release'), in(variables['Build.Reason'], 'PullRequest', 'IndividualCI', 'BatchedCI', 'BuildCompletion'))
         inputs:
-          script: yarn install && yarn build
+          script: yarn install && yarn build:production
```

Comment:
I think @thornbill mentioned we should remove the build command here since yarn install will do that for us.

Reply:
Tested locally to confirm and update pushed